js/templates/modals/orderDetail/summaryTab/orderDetails.html

```diff
@@ -50,7 +50,7 @@ <h2 class="tx4 margRTn"><%= ob.polyT('orderDetail.summaryTab.orderDetails.headin
           <% if (ob.order.shipping && ob.order.shipping.country !== 'NA') { %>
             <div><%= ob.order.shipping.shipTo %></div>
             <% if (ob.order.shipping.address) { %>
-            <div><%= ob.order.shipping.address %></div>
+            <div><%= ob.order.shipping.address.replace(/, /g, ',<br>') %></div>
```

Comment:
Would we want to remove the `,` from before the line break? Otherwise it's leaving a trailing comma at the end of the lines:

![image](https://user-images.githubusercontent.com/794517/29084234-37ea8dde-7c20-11e7-85ff-a4faec6c78e3.png)